src/components/Subtitle/SubtitleStream.scss

```
@@ -0,0 +1,72 @@
+
+      // Fade the top of the band so any partially-clipped character row
+      // melts into the background instead of getting cut in half. The
+      // fade height matches one line of text (line-height: 1.4em).
+      mask-image: linear-gradient(to bottom, transparent 0, black 1.4em, black 100%);
+      -webkit-mask-image: linear-gradient(to bottom, transparent 0, black 1.4em, black 100%);
+
+      p {
+        margin: 0;
+        line-height: 1.4;
+        overflow-wrap: anywhere;
+        word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)